ERCS/erc-XXXX.md

```diff
@@ -13,60 +13,64 @@ requires: 712, 1193
 
 ## Abstract
 
-A new `secureRequest` method in addition to EIP-1193 `request` for dApp to interact with extension wallets in a tamperproof way, preventing any MITM from modifying the payload sent to the wallet.
-This will allow for dApps to sign their interaction with EIP-1193, and for wallet to visibly display secure interaction, therefor improving the end user experience by explicitly displaying safety
-visual cues such as a padlock.
+Introduces a new RPC method `wallet_signedRequest` that enables dapps to interact with wallets in a tamperproof manner via "signed requests".
+To send a signed request, the dapp first publishes a set of public keys via its DNS records.
+Then, the dapp signs its plaintext RPC request with a corresponding private key,
```

Comment:
nit: s/The, the dapp signs/The dapp then signs/ ?

Reply:
[7e72a2e](https://github.com/uni-guillaume/ERCs/pull/2/commits/7e72a2eb1cac851a12f74e4d77f9eb90a598a2c3)